scripts/add_mbedtls_sources.py

```
@@ -0,0 +1,45 @@
+# Pulls all mbedTLS sources from arduino-pico's pico-sdk into the build so the
+# firmware can call mbedtls_* APIs (the precompiled arduino-pico libs only
+# expose BearSSL; mbedTLS is shipped as source under lib/mbedtls/library/*.c).
+#
+# Wired in via `extra_scripts = pre:scripts/add_mbedtls_sources.py` on the
+# variants that define HAS_ETHERNET_TLS_API. POSIX-only code paths inside
+# mbedtls are neutralized by src/mbedtls_user_config.h (referenced via the
+# variant's -DMBEDTLS_USER_CONFIG_FILE build flag). Unused symbols are dropped
+# at link time, so non-TLS envs that pull this script in pay nothing.
+
+Import("env")
```

Flake8 (7.3.0)

[error] undefined name 'Import'

(F821)